modular_sand/code/game/objects/items/fleshlight.dm

```diff
@@ -120,51 +120,58 @@
 	var/target_message = ""
 	var/user_lust_amt = NONE
 	var/target_lust_amt = NONE
-	var/obj/item/organ/genital/P
 	var/target
 	var/mob/living/carbon/human/portal_target = ishuman(portalunderwear.loc) && (portalunderwear.current_equipped_slot & (ITEM_SLOT_UNDERWEAR | ITEM_SLOT_MASK)) ? portalunderwear.loc : null
-	var/obj/item/organ/genital/penis/T = portal_target.getorganslot(ORGAN_SLOT_PENIS)
```

Comment:
why the removal of these two?

Reply:
I think they were also used for length and the like, as well as maybe something cum related down the line, unsure